mediawiki/Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN echo http://mirror.yandex.ru/mirrors/alpine/v3.9/community >> /etc/apk/repos
  && cd /var/www/w/extensions \
  && git clone -b $MEDIAWIKI_VERSION --depth 1 https://gerrit.wikimedia.org/r/p/mediawiki/extensions/Cite.git Cite \
  && git clone -b $MEDIAWIKI_VERSION --depth 1 https://gerrit.wikimedia.org/r/p/mediawiki/extensions/SyntaxHighlight_GeSHi.git SyntaxHighlight_GeSHi \ 
+ && git clone -b $MEDIAWIKI_VERSION --depth 1 https://github.com/wikimedia/mediawiki-extensions-MobileFrontend.git MobileFrontend \
```

Comment:
There are 2 examples above how to clone extensions. Use `https://gerrit.wikimedia.org/r/p/mediawiki/extensions/MobileFrontend.git`.

Reply:
Stick to the alphabetical order.